src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -153,16 +156,53 @@ export default {
     },
     suffixedBarcodes() {
```

Comment:
This method although working effectively could benefit from a refactor. I don't think that is necessary at this stage as it would be better to get it into UAT and we will make further changes when we add plate support.

I would however recommend adding some comments to explain what is going on as it is fairly complex.

Reply:
Added some comments. Maybe will refactor later when adding plate support.